src/sync/dependabot-alerts.ts

```diff
@@ -30,12 +30,25 @@ export const getDependabotAlertTask = async (
 	const fromDate = messagePayload.commitsFromDate ? new Date(messagePayload.commitsFromDate) : undefined;
 	const smartCursor = new PageSizeAwareCounterCursor(cursor).scale(perPage);
 
-	const { data: dependabotAlerts } = await gitHubClient.getDependabotAlerts(repository.owner.login, repository.name, {
-		per_page: smartCursor.perPage,
-		page: smartCursor.pageNo,
-		sort: "created",
-		direction: SortDirection.DES
-	});
+	let dependabotAlerts: DependabotAlertResponseItem[];
+	try {
+		const response = await gitHubClient.getDependabotAlerts(repository.owner.login, repository.name, {
+			per_page: smartCursor.perPage,
+			page: smartCursor.pageNo,
+			sort: "created",
+			direction: SortDirection.DES
+		});
+		dependabotAlerts = response.data;
+	} catch (err) {
+		if (err.cause?.response?.status == 403 && err.cause?.response?.data?.message?.includes("Dependabot alerts are disabled for this repository")) {
```

Comment:
Shouldn't it be mapped to a specific GitHubClientError here https://github.com/atlassian/github-for-jira/blob/d0539da1759e90269404c7722e1a4d7b0da962be/src/github/client/github-client-interceptors.ts#L96-L96 ?